Tasks/AzureCLIV2/task.json

```diff
@@ -39,11 +39,11 @@
                 "azureSubscription"
             ],
             "type": "connectedService:AzureRM",
-            "label": "Azure subscription",
+            "label": "Azure Resource Manager connection",
             "required": true,
-            "helpMarkDown": "Select an Azure resource manager subscription for the deployment",
+            "helpMarkDown": "Select an Azure Resource Manager service connection for the deployment",
             "properties": {
-                "EndpointFilterRule": "ScopeLevel != ManagementGroup"
+                "EndpointFilterRule": "ScopeLevel >= Subscription"
```

Comment:
Should we not just remove the filter rule? Even a SPN with RG scope can run cli commands which are specific to that RG. No?

Reply:
Filter has default value of not lettting management group. We can set it empty